.github/workflows/test.yml

```
@@ -10,10 +10,10 @@ jobs:
       matrix:
         python-version: [ "3.10", "3.11", "3.12"]
         requirements-file: [
-          dj42_cms41.txt,
-          dj50_cms41.txt,
-          dj51_cms41.txt,
-          dj52_cms41.txt
+          dj42_cms50.txt,
+          dj50_cms50.txt,
+          dj51_cms50.txt,
+          dj52_cms50.txt
         ]
         os: [
           ubuntu-20.04,
```

```suggestion
          ubuntu-latest,
```
`ubuntu-20.04` has been sunset...

---

LGTM